webapp/channels/src/components/post_view/reaction_list/reaction_list.tsx

```diff
@@ -152,7 +150,7 @@ export default class ReactionList extends React.PureComponent<Props, State> {
         }
 
         let emojiPicker = null;
-        if (this.props.canAddReactions) {
+        if (this.props.canAddReactions && this.state.emojiNames?.length < 40) {
```

Comment:
Would you mind making a constant for this value in `mattermost-redux/src/constants/emojis` to make it easier to find and so that we can use the same value in other places in the code?

Reply:
Done